src/core/store/requests/requestsEnum.ts

```diff
@@ -0,0 +1,10 @@
+export const REQUESTS_ENUM = Object.freeze({
+    getEmployeesList: 'GET_EMPLOYEES_LIST',
+    getEmployee: 'GET_EMPLOYEE',
+    addEmployee: 'ADD_EMPLOYEE',
+    editEmployee: 'EDIT_EMPLOYEE',
+    deleteEmployee: 'DELETE_EMPLOYEE',
+    getSkills: 'GET_SKILLS',
+    getRoles: 'GET_ROLES',
+    getDepartments: 'GET_DEPARTMENT',
+});
```

Comment:
What's you opinion about using `Typescript Enums` instead of this ?

```suggestion
export enum RequestsEnum {
    GetEmployeesList = 'GET_EMPLOYEES_LIST',
    GetEmployee = 'GET_EMPLOYEE',
    AddEmployee = 'ADD_EMPLOYEE',
    EditEmployee = 'EDIT_EMPLOYEE',
    DeleteEmployee = 'DELETE_EMPLOYEE',
    GetSkills = 'GET_SKILLS',
    GetRoles = 'GET_ROLES',
    GetDepartments = 'GET_DEPARTMENT',
}
```